multidict/_pair_list.c

```diff
@@ -93,19 +93,17 @@ ci_key_to_str(PyObject *key)
 {
     PyTypeObject *type = Py_TYPE(key);
     if ((PyObject *)type == _istr_type) {
-        // replace with direct .canonical attr
-        return _PyObject_CallMethodId(key, &PyId_title, NULL);
+        return PyObject_CallMethod(key, "lower", NULL);
     }
     if (PyUnicode_Check(key)) {
-        return _PyObject_CallMethodId(key, &PyId_title, NULL);
+        return PyObject_CallMethod(key, "lower", NULL);
```

Comment:
Please keep it.
Passing `char *` is slow. For working in `LIMITED_API` boundaries we need to pass pre-existing `PyObject *` but I don't care at the moment.